<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LP-TZ-0X107825" failures="0" tests="5" name="com.tpo.test.TestAPI" time="2.947" errors="1" timestamp="2023-02-21T19:26:34 IST" skipped="0">
  <testcase classname="com.tpo.test.TestAPI" name="testGetMethod" time="0.140"/>
  <system-out/>
  <testcase classname="com.tpo.test.TestAPI" name="closeDriver" time="0.000">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.close()&quot; because &quot;com.tpo.testbase.TestBase.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.close()" because "com.tpo.testbase.TestBase.driver" is null
at com.tpo.testbase.TestBase.closeDriver(TestBase.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:404)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- closeDriver -->
  <system-out/>
  <testcase classname="com.tpo.test.TestAPI" name="testPostMethod" time="1.556"/>
  <system-out/>
  <testcase classname="com.tpo.test.TestAPI" name="testDeleteMethod" time="0.627"/>
  <system-out/>
  <testcase classname="com.tpo.test.TestAPI" name="testPutMethod" time="0.624"/>
  <system-out/>
</testsuite> <!-- com.tpo.test.TestAPI -->
